src/diffusers/models/__init__.py

```diff
@@ -85,7 +86,7 @@
         from .transformers import (
             DiTTransformer2DModel,
             DualTransformer2DModel,
-            HunyuanDiT2DModel,
```

Comment:
The `HunyuanDiT2DModel` shouldn't be removed here.

Reply:
I added it again.